### ISSUE TITLE
Receiving  init() missing 1 required positional argument: ‘certification_country’ when first starting Kodi

### ISSUE THREAD

**Issue by kpmelvin** [croneter/PlexKodiConnect#1657]
I'm getting the above error when first starting Kodi. It's a fresh install of Kodi 19.2 and PlexKodiConnect Beta v3.5.3.
[kodi.log](https://github.com/croneter/PlexKodiConnect/files/7325223/kodi.log)

Providing updated log. It appears the error happens right after various media scanners initialize. 

Excerpt,
2021-10-12 07:20:57.405 T:11612   DEBUG <general>: MUSIC_INFO::CMusicInfoScanner::Process - Starting scan
2021-10-12 07:20:57.406 T:7856    ERROR <general>: PLEX.utils: Error encountered:  - __init__() missing 1 required positional argument: 'certification_country'

..............................

2021-10-12 07:20:57.411 T:7856    ERROR <general>: PLEX.utils:         return tmdb.TMDBMovieScraper(settings, language, certcountry)
2021-10-12 07:20:57.411 T:7856    ERROR <general>: PLEX.utils:     TypeError: __init__() missing 1 required positional argument: 'certification_country'
............................

[kodi.log](https://github.com/croneter/PlexKodiConnect/files/7330994/kodi.log)

**Comment by attinderdhillon** [comment]
I have the same issue on latest Beta.

**Comment by Glassertje** [comment]
Also happens on the matrix stable version

**Comment by jun6lee** [comment]
Seeing this a lot. I have a lot of foreign media, may be connected?!

**Comment by kpmelvin** [comment]
I'm running on Windows 10. Attached is a copy of my tmdb;py file.
[tmdb.zip](https://github.com/croneter/PlexKodiConnect/files/7349572/tmdb.zip)
.

**Comment by croneter** [comment]
Here's the culprit:
Original `metadata.themoviedb.org.python`-code
```python
class TMDBMovieScraper(object):
    def __init__(self, url_settings, language, certification_country):
        self.url_settings = url_settings
        self.language = language
        self.certification_country = certification_country
        self._urls = None
```
The code you guys got:
```python
class TMDBMovieScraper(object):
    def __init__(self, url_settings, language, search_language, certification_country):
        self.url_settings = url_settings
        self.language = language
        self.search_language = search_language
        self.certification_country = certification_country
        self._urls = None
```
So you seem to be using this fork: https://github.com/mkaflowski/metadata.themoviedb.org.python instead of the "original" Kodi https://github.com/xbmc/metadata.themoviedb.org.python. 

I know what to do to fix this, but it'll need some time.

**Comment by Glassertje** [comment]
I just checkt in kodi, and i have indeed that code, but it is standard kodi install, and the plugin is from kodi team. Did not change anything. No idea how it is mixed up.

**Comment by mkaflowski** [comment]
I have added default value: https://github.com/mkaflowski/metadata.themoviedb.org.python/blob/master/python/lib/tmdbscraper/tmdb.py
Is that going to help?

**Comment by kpmelvin** [comment]
> 
> 
> I have added default value: https://github.com/mkaflowski/metadata.themoviedb.org.python/blob/master/python/lib/tmdbscraper/tmdb.py Is that going to help?

I'm baffled  as to how the fork got installed. In any case I pasted your change into the tmdb.py file. It worked, I no longer get the error when I start Kodi.

FWIW I checked the version, it's 1.4.2 by Team Kodi.

**Comment by boardlord1** [comment]
> 
> 
> > I have added default value: https://github.com/mkaflowski/metadata.themoviedb.org.python/blob/master/python/lib/tmdbscraper/tmdb.py Is that going to help?
> 
> I'm baffled as to how the fork got installed. In any case I pasted your change into the tmdb.py file. It worked, I no longer get the error when I start Kodi.
> 
> FWIW I checked the version, it's 1.4.2 by Team Kodi.

Indeed, by replacing tmdb.py with the version mkaflowski linked to, the error is gone. Thanks!

**Comment by Glassertje** [comment]
I can confirm, by replacing it, the error is gone.

**Comment by mkaflowski** [comment]
You can just download zip from my fork. I have made pull request and hope that it will be in official channel but I am little confused where should I commit because there are 2 repositories.

**Comment by croneter** [comment]
Please try PKC version 3.5.4. Hopefully fixes this easily...

**Comment by kpmelvin** [comment]
@croneter confirmed as well, your update fixed the issue when using the 1.4.2 version of tmdb.py

much appreciated

**Comment by croneter** [comment]
Thanks for reporting back!
